src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public abstract String doWork();
+
```

Comment:
Wrong method signature. 

Create class Machine containing methods public abstract `void` doWork() and public abstract `void` stopWork().